### DISCUSSION TITLE
wiswiki and google groups retirement

### DISCUSSION THREAD

**Discussion by efucile** [wmo-im/iwxxm#262] [A-Team business]
@wmo-im/tt-avxml-developers wiswiki will be retired from 1st June. A static copy is already available at [http://old.wmo.int/wiswiki/wiswiki.wmo.int/tiki-index.html] . We need to
1. check that what we need is in the static copy
1. decide where to move the pages that cannot be changed any more. A candidate is [https://community.wmo.int/activity-areas/wmo-codes] 

Also google groups will be switched off during summer. Can we use gitHub as replacement?

**Comment by blchoy** [comment]
There are two things in existing wiswiki which we would like to maintain, viz. description of IWXXM (supposed to be more official than those in GitHub) and meeting minutes.  [WMO Codes](https://community.wmo.int/activity-areas/wmo-codes) could be a good place as the description of IWXXM can go into "WMO CODES" while the meeting minutes can go into "COMMUNITY".

![image](https://user-images.githubusercontent.com/30064374/79525140-bcd7dd00-8094-11ea-85ee-519f9e440f61.png)

I am a bit concerned for loosing the Google Group, as it will mess up our GitHub Issues which we wish to be used for confirmed bugs/enhancements or significant discussion items, for example with questions like "How can I...".

**Comment by mgoberfield** [comment]
Ref: Google Groups closure.

I second Choy's concern here.  There should be a listserver or forum where anyone can ask IWXXM-related questions or raise concerns, analogous to "Tier 1 support." Issues that cannot be resolved at that level may be escalated into GitHub issues.

**Comment by moryakovdv** [comment]
@mgoberfield, same here.
BTW, the first place I searched and posted information about IWXXM was that Google group. I suspect, I'm not alone. So we should provide good replacement and **advert** it among others.

**Comment by blchoy** [comment]
During a conversation last Friday Enrico asked what items on wiswiki should be moved (or be linked from) the new website.  I think the previous meeting notes are a must.  We may want to keep those brief descriptions may be in the form of README.md.  Tutorials in my opinion should be merged with the TAC-to-XML_Guidance for form a separate document or Wiki similar to the [AIXM Coding Guidelines](https://ext.eurocontrol.int/aixm_confluence/display/ACGAIP/Coding+Guidelines).

**Comment by blchoy** [comment]
I have a quick chat with XiaoXia of WMO on 3 Aug 2020 and the following are the actions to be taken:

1. WMO IT guys will fix the incorrect links on the WISWiki archive so that we could retrieve previous materials
2. XiaoXia showed me the editing interface of the webpages on the Community Platform which is exactly the same as the previous one for WISWiki (!) In any case the team will do some editing on the retrieved materials and pass XiaoXia HTML files for her to cut-and-paste onto the Community Platform.
3. For the minutes of previous meetings I think there is nothing requiring the team's attention for migration.  I presume WMO can move it to a new place in bulk.
4. For the Google group a few of those extremely important few ones will be retained as far as I understand but ours are definitely not among them.

So for this issue I think we may need to create a job for (2) for the team.  We may also need continue seeking a solution for (4).

**Comment by blchoy** [comment]
Seems to me that the stuff on WISWiki is already history.  If they will remain in the archive forever (e.g. meeting notes) there is no point to move to the new place.  @amilan17 and I will work on a new team's page on WMO Community Website and may borrow some of the materials there, but as time progresses most items previously on WISWiki (like tutorial) do not seem to have a value to be maintained.

Views welcomed.
